app/controllers/conversations.js

```diff
@@ -144,9 +216,27 @@ class ConversationsController extends BaseController {
               conversation_id: conversationId,
             }))
           ) {
-            await obj.save();
+            await participant.save();
+            participants.push(participant.params._id);
           }
         }
+
+        if (participants.length) {
+          const currentUserLogin = (await User.findOne({ _id: currentUserId }))
+            ?.params?.login;
+          await this.#notifyAboutConversationCreate(
+            ws,
+            requestId,
+            conversation,
+            {
```

Comment:
also we can move this object formation inside `notifyAboutConversationCreate`, and here pass a login only